readthedocs/projects/tasks.py

```diff
@@ -1272,7 +1268,7 @@ def fileify(version_pk, commit, build):
     This is so we have an idea of what files we have in the database.
     """
     version = Version.objects.get_object_or_log(pk=version_pk)
-    if not version:
+    if not version or version.type == EXTERNAL:
```

Comment:
Not sure where we are with indexing or not external versions p:

Reply:
We aren't indexing external versions.